server/cssfile.js

```diff
@@ -62,10 +59,11 @@ CssFile.prototype.setContent = function(source, callback){
 		return;
 	}
 
-	this.parsed.stylesheet.rules.forEach(function(rule){
-		var position = rule.position;
-		position.start.line--;
-		position.start.column--;
+	this.parsed.nodes.forEach(rule => {
+		var source = rule.source;
+		source.start.line--;
+		source.start.column--;
+		source.end.column++;
```

Comment:
I didn't fully understand why this was done previously so I made this equivalent.